languages/go/tests/other/test.polar

```diff
@@ -0,0 +1 @@
+f();
```

Comment:
no reason; it doesn't matter that it's different (see https://github.com/osohq/oso/pull/1247#discussion_r722500705), but there's no reason it can't be the same. will update

Reply:
9c84074